mobile/src/libs/hooks/use-app-form/use-app-form.hook.ts

```diff
@@ -42,14 +45,17 @@ const useAppForm = <T extends FieldValues = FieldValues>({
   }
 
   const {
+    watch,
     control,
     handleSubmit,
-    formState: { errors },
+    formState: { errors, isValid },
```

Comment:
```suggestion
    formState: { errors },
```
We don't need it, we can just check for the presence of errors

Reply:
@fshabanov i tried to change isValid to errors, but errors is not working properly as isValid does